### PR TITLE
Add form-checkbox to travis-form

### PR DESCRIPTION
**Pull request by artursmirnov**
This is a part of giant https://github.com/travis-ci/travis-web/pull/1949/ that is quite independent, so I've extracted it to a separate PR. Here's the link to design in scope of [MOSS] Migrate tab: https://github.com/travis-pro/team-teal/issues/2915#issuecomment-465244581

### CODE REVIEW

**Comment by artursmirnov** [comment]
Hey @sorta 👋I have fixed tests, so it's ready for review, please take a look once you have a moment.

**Comment by sorta** [comment]
Hi @artursmirnov ! Sure thing, I'll take a look :)

File: public/images/svg/checkbox-unchecked.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Build-Stage-Passed" x="0px" y="0px" viewBox="0 0 15 15" style="enable-background:new 0 0 15 15;" xmlns="http://www.w3.org/2000/svg">
+  <style type="text/css">
+	.st0{fill:none;stroke:#3EAAAF;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;}
+</style>
+  <g id="stage-passed-icon">
+    <circle class="st0" cx="7.5" cy="7.5" r="7" style="stroke: rgb(157, 157, 157);"/>
```
Comment (sorta) [inline public/images/svg/checkbox-unchecked.svg]: The stroke style on this element could be moved into the <style> tag above, replacing the existing declaration on the `.st0` class.

Comment (artursmirnov) [reply public/images/svg/checkbox-unchecked.svg]: Thanks for the catch! I didn't notice it, seems like some editor have put it there 🤔And RGB format doesn't make any sense too.

Comment (sorta) [reply public/images/svg/checkbox-unchecked.svg]: Heheh yeah absolutely. I've definitely had SVG editors do some weird things. And yeah, probably no reason to have it in RGB

**Pull request merged**